lib/serializables/log-event.js

```diff
@@ -32,7 +32,18 @@ function LogEventFactory (
     pretty,
     console
 ) {
+    //default turn off the colorful output to achieve tidy and clean log file.
+    //however user can still enable it via command line argument.
+    var colorEnable = false;
+    var argv = process.argv;
```

Comment:
@heckj : Are you suggesting designing a function (such as **setColorEnable**) in LogEvent. Then in each app's index.js (on-http/on-taskgraph/on-dhcp-proxy/on-syslog/on-tftp), call this function to set color mode bases on command line options)?
 
I agree bury those command line options in the LogEvent is not a good idea. But to avoid cross repositories change and multi code copying, how about define a common argv processor in on-core then execute the common processor in on-core index.js or call the common argv processor in each app index.js.